essay.tex

```diff
@@ -0,0 +1,61 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{The impacts of Agile Methodologies on the motivation within a team across a projects.}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{Student Number - 1507729}
+
+\begin{document}
+
+\maketitle
+
+\abstract The Agile development methodology is the most prominent within the software development. It is at it's core designed to be Agile and to be adaptable for any and every team following it's guidelines. However that can also lead teams failing to fully utilise the Agile methodology. This paper will explore how the Agile method can impact the motivation of teams and the impact it has had on some existing teams.
```

Comment:
Clean up the sentences, e.g: `At it's core, agile is designed to be adaptable...`